styles.css

```diff
@@ -0,0 +1,78 @@
+html,
+body {
+  height: 100%;
+  font-family: 'Open Sans', sans-serif;
+  text-align: center;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  height: 475px;
+  width: 725px;
+  box-sizing: border-box;
+}
+
+.profile {
+  display: inline-block;
+  height: 415px;
+  width: 313px;
+  margin: 15px;
+  border: 2px solid #c9c9c9;
+  border-top: 0px;
+  border-radius: 10px;
+  padding-bottom: 10px;
+}
+
+.pic {
+  width: 315px;
+  height: 250px;
+  border: 0px;
+  margin: 0 -2px 0 -3px;
+  border-radius: 8px 8px 0px 0px;
```

Comment:
when doing `0` anything, you don't need the `px` specification. `0` means the same in `em`, `rem`, `pt`, etc.
```css
border-radius: 8px 8px 0 0
```